header-validator/validate-os.js

```diff
@@ -77,9 +92,13 @@ function validate(str, paramChecks) {
 }
 
 export function validateRegisterOsSource(str) {
-  return validate(str, {})
+  return validate(str, {
+    'debug-reporting': optional(bool),
+  })
 }
 
 export function validateRegisterOsTrigger(str) {
-  return validate(str, {})
+  return validate(str, {
+    'debug-reporting': optional(bool),
+  })
```

Comment:
Separately, would you mind merging these two functions since they are identical?

Reply:
Merged, thanks.